scaffolds/app_engine/app/static/main.js

```diff
@@ -0,0 +1,51 @@
+async function queryApi(prompt) {
```

Comment:
Add a comment at the top of this file to describe what this file is doing. Since there's so many files it'll be good to know what the purpose of this file is.

Reply:
Done!